client/internal/packagessyncer.go

```diff
@@ -20,6 +21,7 @@ type packagesSyncer struct {
 	sender            Sender
 
 	statuses *protobufs.PackageStatuses
+	mut      *sync.Mutex
```

Comment:
Done 👍 (although as a personal preference I prefer the `mut` shorthand to avoid confusion with mux in context of an HTTP multiplexer, but we wouldn't worry about that here)